pkg/apis/org/v1/che_types.go

```diff
@@ -143,6 +147,8 @@ type CheClusterSpecAuth struct {
 	OauthSecret string `json:"oAuthSecret"`
 	// KeycloakImage is image:tag used in Keycloak deployment
 	KeycloakImage string `json:"identityProviderImage"`
+	// KeycloakImagePullPolicy is the image pull policy used in Keycloak registry deployment: default value is Always
+	KeycloakImagePullPolicy corev1.PullPolicy `json:"identityProviderImagePullPolicy"`
```

Comment:
it's too bad we have Keycloak* varibles mixed in with identityProvider* images but that's a refactoring for another time :)

Reply:
Yeah, the problem here is that it would be a breaking change in the API. So we would have to manage several versions of the API, which is not straightforward afaik.